flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/batch/connectors/hive/HiveTableSink.java

```diff
@@ -161,4 +164,37 @@ private String toStagingDir(String finalDir, Configuration conf) throws IOExcept
 		fs.deleteOnExit(path);
 		return res;
 	}
+
+	@Override
+	public List<String> getPartitionFieldNames() {
+		return catalogTable.getPartitionKeys();
+	}
+
+	@Override
+	public void setStaticPartition(Map<String, String> partitions) {
```

Comment:
Variable name "partitions" might be a little confusion. It appears to be a partition spec, with partition column names and their values. Consider renaming it.

Reply:
OK will do. Actually the variable name is inherited from the interface. Perhaps it needs renaming there as well.